packages/app/src/pages/Rewards/index.tsx

```diff
@@ -0,0 +1,36 @@
+// Copyright 2025 @polkadot-cloud/polkadot-staking-dashboard authors & contributors
+// SPDX-License-Identifier: GPL-3.0-only
+
+import { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { PageTitle } from 'ui-core/base'
+import { Active } from './Active'
+import { PayoutHistory } from './PayoutHistory'
+import { Wrapper } from './Wrappers'
+
+export const Rewards = () => {
+  const { t } = useTranslation('pages')
+  const [activeTab, setActiveTab] = useState<number>(0)
+
+  return (
+    <Wrapper>
+      <PageTitle
+        title={t('rewards.rewardsCalculator')}
+        tabs={[
+          {
+            title: t('rewards.calculator'),
```

Comment:
This page can be `Overview`. The payout history graph currently present in "Payout History" tab can be displayed at this top of this page, with the new "Projected Rewards" module being displayed below it.

The Payout History tab can then just list historical payout records.